esercizio1.html

```diff
@@ -6,45 +6,44 @@
     <title>Esercizio1</title>
 
     <link rel="stylesheet" href="style.css">
-
+    
 </head>
 
-<body style="background-image: linear-gradient(to right,rgb(73, 4, 119),violet); margin:0; padding:0; height: 100%;">
+<body>
 
     <div class ="main">
 
-        <div class="intestazione">
-               
-                <h1 ><b>Duis mollis,<br>est non<br>commodo<br>luctus.</b></h1>
-
-                <p>Nulla vitae elit libero, a<br><br>pharetra augue. Lorem ipsum<br><br>dolor sit amet, consectetur.</p>
-        
+        <div class="intestazione flex_elements">
+            <div style="width: 180px;">
+                <div>
+                <h1 ><b>Duis mollis, est non commodo luctus.</b></h1>
+                </div>
+                <div style="line-height: 25px;">
+                <p>Nulla vitae elit libero, a  pharetra augue. Lorem ipsum  dolor sit amet, consectetur.</p>
+                </div>
+            </div>
         </div>
 
-
-        <form class="login">
-
-<<<<<<< HEAD
-            
-=======
->>>>>>> 0bfca1cc65f6eb05cfb874e263464908aeea397e
-            <label><b>EMAIL</b></label>
-            <br><br>
-            <input type="text" name="email">
-            <hr>
-            <br><br>
-            <label><b>PASSWORD</b></label>
-            <br><br>
-            <input type="text" name="password">
-            <hr>
-            <br>
-
-            <button type="button" id="btn1" ><b>Sign Up</b></button>
-            <br>
-            <hr>
-            <button type="button" id="btn2" >Forgot Password?</button> 
-
-        </form>
+        <div class="login flex_elements">
```

Comment:
per quanto riguarda il css c'è un selettore che permette di selezionare i 'figli' di un elemento ( un figlio è un tag racchiuso in un altro tag)
<div>
    <div></div> // questo div è figlio del div che lo wrappa ( il div più esterno )
</div>

tu qui hai dato una classe al div <div class="login flex_elements"> che sarebbe il contenitore, e poi hai dato a tutti i div figli una classe flex_elements

con il css puoi fare che 
.login .flex_elements > div {
   // in questo caso il css che specifichi qui verra applicato a tutti i div (figli diretti) contenuti nel tag padre
}

in questo modo non serve specificare la classe flex_elements nei div figli